portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Olu's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+      <div class="row">
+        <img src="images/logo.png"
+    </div>   
+    <ul class="main-nav">    
+        <li><a href="index.html" style="color:white"> HOME </a></li>
+        <li><a href="gallery.html" style="color:white"> PHOTOGRAPHY </a> </li>
+		<li><a href="" style="color:white"> PROJECT SHOWCASE </a></li>
+        <li><a href="index2.html" style="color:white"> ABOUT </a></li>
+        <li class="active"><a href="contact.html" style="color:white"> CONTACT </a></li>
+    </ul>
+<h1 style=margin-top:50px><i>Let's Connect!</h1>
+
+<form action>
+  <label for="name">Name:</label>
+  <input type="text" id="name" name="name"><br><br>
+  <label for="email">E-Mail:</label>
+  <input type="text" id="email" name="email"><br><br>
+  <input type="submit" value="Submit">
+</form>
+</body>
+  <footer>
+	&copy; Copyright 2020 Olus's Website
+</footer> 
```

Comment:
Let's fix the indentation here and in the other HTML files so things are consistently formatted and spaced.

Reply:
fixed